test_scripts/Protocol/commonProtocol.lua

```diff
@@ -80,10 +99,18 @@ function common.startServiceProtectedNACK(pAppId, pServiceId, pRequestPayload, p
     end)
 end
 
-
-function common.startServiceUnprotectedNACK(pAppId, pServiceId, pRequestPayload, pResponsePayload)
+function common.startServiceUnprotectedNACK(pAppId, pServiceId, pRequestPayload, pResponsePayload, pExtentionFunc)
+    if pExtentionFunc then pExtentionFunc() end
     local mobSession = common.getMobileSession(pAppId)
-    mobSession:StartService(pServiceId, bson.to_bytes(pRequestPayload))
+    local msg = {
+        serviceType = pServiceId,
+        frameType = constants.FRAME_TYPE.CONTROL_FRAME,
+        frameInfo = constants.FRAME_INFO.START_SERVICE,
+        sessionId = mobSession,
+        encryption = false,
+        binaryData = bson.to_bytes(pRequestPayload)
+    }
+    mobSession:Send(msg)
```

Comment:
`StartService` function contains `StartService` request and also the expectation of `StartServiceACK` so that's why we can not use `StartService` function here.

Reply:
@GetmanetsIrina By that logic, `startServiceProtectedNACK` should probably not be using the `mobSession:StartSecureService` function, but it does so without issue as of now. That said, this is a minor detail, and the tests appear to be working as-is, so I will mark this as resolved